doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.4.99-cs3_to_2.7_application.rst

```diff
@@ -74,6 +74,38 @@ General
 
      The previously used SOC1-based board files have been removed.
 
+nRF Desktop
+-----------
+
+The following changes have to be made for :ref:`nrf_desktop` application to work in the same way as in the release v2.4.99-cs3.
+
+* Due to the migration to the new hardware model, commonly referred as "hardware model v2", the board target of the nRF54H20 DK has changed to the ``nrf54h20dk/nrf54h20/cpuapp``.
+  Building command for application has changed and it is now:
+
+  .. code-block:: console
+
+    west build -b nrf54h20dk/nrf54h20/cpuapp
+
+* Radio core now uses the :ref:`ipc_radio` application from ``sdk-nrf`` instead of the Bluetooth HCI RPMsg sample from ``sdk-zephyr``.
```

Comment:
Link to the Zephyr BT sample? Is it this one https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/samples/bluetooth/hci_ipc/README.html?

Reply:
Yes, how to link it?